internal/config/provider.go

```diff
@@ -108,25 +112,34 @@ func UpdateProviders(pathOrUrl string) error {
 		return fmt.Errorf("failed to save providers to cache: %w", err)
 	}
 
-	slog.Info("Providers updated successfully", "count", len(providers), "from", pathOrUrl, "to", cachePath)
+	slog.Info("Providers updated successfully", "count", len(providers), "from", pathOrURL, "to", cachePath)
 	return nil
 }
 
 func Providers(cfg *Config) ([]catwalk.Provider, error) {
 	providerOnce.Do(func() {
+		start := time.Now()
+		defer func() { slog.Info("Catwalk update took " + time.Since(start).String()) }()
```

Comment:
please remove this logic (update time log), is only valuable when debugging for the changes you did but in overral isn't a log that matters to keep